Source/gui/SNTBinaryMessageWindowView.swift

```diff
@@ -57,7 +57,7 @@ import santa_gui_SNTMessageView
 }
 
 func copyDetailsToClipboard(e: SNTStoredEvent?, customURL: String?) {
-  var s = "Santa blocked \(e?.fileBundleName ?? "an application:")"
+  var s = "Santa blocked \((e?.fileBundleName?.isEmpty == false) ? e!.fileBundleName! : "an application")"
```

Comment:
With the change to SNTFileInfo, the previous behavior or just nil-checking fileBundleName should be enough, no?

Reply:
Yeah, it would be enough today. I liked the idea of matching the logic that was used to print the "Application"/"File" label. It felt odd to make that check less robust given this code is reliant on data from another component, so I made these checks more robust so that things are more consistent.